deployment/terraform/variables.tf

```diff
@@ -58,6 +58,8 @@ variable "google_geocoding_api_key" {}
 
 variable "rollbar_server_side_access_token" {}
 
+variable "react_app_rollbar_post_client_item_access_token" {}
```

Comment:
I think that it would be good to consider renaming this Terraform variable to something like `rollbar_client_side_access_token` for consistency with `rollbar_server_side_access_token`. We can leave the environment variable itself with the `REACT_APP_` prefix.